Aptoide/aptoide-client-v8
# [ASV-1893] Remove billing

**What does this PR do?**

This PR aims to remove everything related to billing from the app;

**Database changed?**

   Yes

**How should this be manually tested?**

Test the flows related to downloads and make sure everything is working fine. If you know of any apps that were paid in the past, make sure everything works as intended in those cases;

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1893](https://aptoide.atlassian.net/browse/ASV-1893)

**Code Review Checklist**
